apps/parse/lib/simple_csv.ex

```diff
@@ -3,8 +3,6 @@ defmodule SimpleCSV do
   A simpler, but higher-performance, CSV parser.  Expects that that lines being
   parsed don't contain any commas or quotes inside quoted fields.
   """
-  @compile :native
```

Comment:
Apparently native compilation is not supported anymore: https://groups.google.com/forum/#!topic/erlang-programming/z7rAKtAaClc